ovs/lib/helpers/toolbox.py

```diff
@@ -197,3 +197,18 @@ def change_service_state(client, name, state, logger=None):
         else:
             logger.debug('  {0:<15} - Service {1} {2}'.format(client.ip, name, action))
             print '  [{0}] {1} {2}'.format(client.ip, name, action)
+
+    @staticmethod
+    def wait_for_service(client, name):
```

Comment:
I'd add a status to this call
Default can be True, so you wait until the service is running
But this option also allows you to wait until a service is stopped entirely